name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dashboard/MaxDrawdownDurationWidget.java

```diff
@@ -69,6 +68,9 @@ public void update(PerformanceIndex index)
         isUntilEndOfPeriod = recoveryTime.getEnd().equals(index.getReportInterval().getEnd());
         String recoveryTimeSupplement = isUntilEndOfPeriod ? Messages.TooltipMaxDrawdownDurationEndOfPeriod
                         : Messages.TooltipMaxDrawdownDurationFromXtoY;
+        
+        indicator.setText(MessageFormat.format(Messages.LabelXDays, maxDDDuration.getDays()
+                        + (isUntilEndOfPeriod ? "+" : ""))); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
The boolean ```isUntilEndOfPeriod``` is assigned twice. At this point in time, it says whether the recovery period is until end of the period, not whether the max drawdown duration is ongoing.